lib/locations/location.rb

```diff
@@ -1,3 +1,19 @@
 module Locations
-  Location = Struct.new(:id, :name, :address, :booking_location_id, :phone, :hours, :twilio_number, :lat_lng)
+  class Location
+    include ActiveModel::Model
```

Comment:
What's the rationale for this? If you just want to support kwarg initialisation maybe just convert the `Struct` to `OpenStruct`? Also for the predicate method you could just pass a block to define methods within the scope of the struct

```ruby
Struct.new(:id, :blah) do
  def online_booking_enabled?
    # ...
  end
end
```